.ci/format.sh

```diff
@@ -32,6 +32,8 @@ format() {
     local allowlist_file="
         src/Equations/elastic/Model/Datastructures.h
         src/Equations/elastic/Model/IntegrationData.h
+        src/Equations/acoustic/Model/Datastructures.h
+        src/Equations/acoustic/Model/IntegrationData.h
```

Comment:
It's best to add the whole folder here (see the first `allowlist`); the `AcousticSetup.h` may also join there.

If there's an issue with the prefixed variable names like `i_material` or the likes—the prefix and the underscore may go in all cases.

(or was there something blocking in that file?)